tests/unittest/_torch/modules/test_fused_moe.py

```diff
@@ -182,10 +182,10 @@ def per_rank_test_fused_moe_alltoall(job_id):
             )
         alltoall_model.to("cuda")
         alltoall_model.load_weights([weights])
-        with mock.patch.object(CutlassFusedMoE,
+        with mock.patch.object(WideEPFusedMoE,
                                "select_alltoall_method_type",
                                return_value=AlltoallMethodType.NotEnabled):
-            ref_model = CutlassFusedMoE(
+            ref_model = WideEPFusedMoE(
```

Comment:
I think the reference model should be `CutlassFusedMoE`.

Reply:
Right now it doesn't really matter because `WideEPFusedMoE` is a superset of `CutlassFusedMoE`. I can change it to `CutlassFusedMoE`.

Reply:
I see. Either is OK now. I can polish the unittest later.

Reply:
Changed ref to `CutlassFusedMoE`